### PR TITLE
Intel ffmpeg patches upstream 2020q3 queue

### PR DESCRIPTION
**Pull request by FocusLuo**
Intel ffmpeg patches upstream 2020Q3 queue

Signed-off-by: Luo Focus <focus.luo@intel.com>

### CODE REVIEW

**Comment by xhaihao** [comment]
could you please provide the FFmpeg baseline in the commit log?

**Comment by FocusLuo** [comment]
> could you please provide the FFmpeg baseline in the commit log?
It is already in the README file.
Intel upstream patches queue for ffmpeg

ffmpeg mainline codebase
commit: e71d73b09652f4fc96e512a7d6d4c2ab41860f27

**Comment by xhaihao** [comment]
FFmpeg e71d73b doesn't support QSV AV1, the patch for QSV AV1 is needed. Or you may change the FFmpeg mainline codebase.

File: README.md [hunk 1]
```diff
@@ -1,2 +1,30 @@
...
+    $ for i in *.patch; do patch -p1 < $i; done
+
+
+Known issues:
+1, ffmpeg-vaapi vp8 decode multi-thread HWAccel decode error
+issue number:
+    https://github.com/intel/media-driver/issues/1028
+    https://github.com/intel/media-driver/issues/317
+ffmpeg patch fixing:
+    https://git.ffmpeg.org/gitweb/ffmpeg.git/commit/eb6bb8f32fdc9be89cce65869dce9dd950e91be2
```
Comment (xhaihao) [inline README.md]: The issue has been fixed in FFmpeg, you'd be better to update the code base.

**Comment by uartie** [comment]
What is the purpose for this new repo?  Why can't you just tag ffmpeg-cartwheel?

**Comment by xuguangxin** [comment]
agree with @uartie 
The  patchset deeply coupled with ffmpeg upstream code.  Use a separate repo may introduce mismatch issue. and maintain the repo will be a big problem. 
Is it possible use the patchset directory in cartwheel directly?

thanks

File: README.md (hunk 1 above)
Comment (FocusLuo) [reply README.md]: Ok, updated

**Comment by FocusLuo** [comment]
> What is the purpose for this new repo? Why can't you just tag ffmpeg-cartwheel?

Aligned with Artie at tomorrow morning discussion to use

**Comment by FocusLuo** [comment]
> What is the purpose for this new repo? Why can't you just tag ffmpeg-cartwheel?

Aligned with Artie at this morning discussion, he agreed on this repo using.

**Comment by FocusLuo** [comment]
> agree with @uartie
> The patchset deeply coupled with ffmpeg upstream code. Use a separate repo may introduce mismatch issue. and maintain the repo will be a big problem.
Artie will have idea on how to auto work on this maintain effort. 

> Is it possible use the patchset directory in cartwheel directly?
In a ffmpeg sub directory maybe is the same thing under the ffmpeg root directory. I am not sure whether there will have any risk in future, I can send an email to ask for 

> 
> thanks

**Pull request merged**